json-path/src/main/java/io/zeebe/msgpack/mapping/MsgPackMergeTool.java

```diff
@@ -57,10 +57,18 @@ public void reset() {
   }
 
   public void mergeDocument(DirectBuffer document, Mapping... mappings) {
+    mergeDocument(document, false, mappings);
+  }
+
+  /**
+   * @param strictMode if true, throws exceptions on ambiguous mapping results; if false, ensures
+   *     that no exceptions are thrown and assumes default values
+   */
+  public void mergeDocument(DirectBuffer document, boolean strictMode, Mapping... mappings) {
```

Comment:
How about mergeDocumentStrict instead of the flag? Is maybe more explicit. The same for the extract method.

Reply:
Hm, it's not really the merge that is strict or not, but the treatment of mapping results. Can't really think of a better name, though.